docs/en/latest/how-to-build.md

```diff
@@ -36,22 +36,22 @@ You can install Apache APISIX in a variety of ways, including source code packag
 You need to download the Apache source release first:
 
 ```shell
-$ mkdir apisix-2.3
-$ wget https://downloads.apache.org/apisix/2.3/apache-apisix-2.3-src.tgz
-$ tar zxvf apache-apisix-2.3-src.tgz -C apisix-2.3
+$ mkdir apisix-2.4
+$ wget https://downloads.apache.org/apisix/2.4/apache-apisix-2.4-src.tgz
+$ tar zxvf apache-apisix-2.4-src.tgz -C apisix-2.4
 ```
 
 Install the Lua libraries that the runtime depends on:
 
 ```shell
-cd apache-apisix-2.3
+cd apache-apisix-2.4
```

Comment:
`apache-apisix-2.4` => `apisix-2.4`

Reply:
Because the `tar zxvf apache-apisix-2.4-src.tgz -C apisix-2.4` command is extracted to the `apisix-2.4` directory.